packages/create-cloudflare/e2e-tests/cli.test.ts

```diff
@@ -80,7 +80,7 @@ describe.skipIf(frameworkToTest || isQuarantineMode())(
 				],
 			});
 
-			expect(projectPath).toExist();
```

Comment:
This uses a custom assertion defined [here](https://github.com/cloudflare/workers-sdk/blob/main/packages/create-cloudflare/e2e-tests/setup.ts#L4). In hindsight, it should probably be named something like `toExistOnDisk`. I think we want to retain this assertion instead of switching to `toBeTruthy`. 

I'm curious why this needed to get changed. Is the setup file causing problems?

Reply:
I've changed it back to `toExist()`. The main issue was TypeScript, since Vitest has changed how to define custom matchers